app/uk/gov/hmrc/gform/formtemplate/SectionHelper.scala

```diff
@@ -29,4 +29,13 @@ object SectionHelper {
 
   def pages(sections: List[Section]): List[Page] =
     sections.flatMap(pages(_).toList)
+
+  def addToListRepeater(section: Section): Option[FormComponent] =
```

Comment:
Just curious why we don't have Section.fold, like we do in other cases?

Reply:
Because backend is not so polished

Reply:
ok